packages/react-composites/src/composites/CallComposite/Lobby.tsx

```diff
@@ -27,9 +27,9 @@ export const Lobby = (props: LobbyProps): JSX.Element => {
   const [isButtonStatusSynced, setIsButtonStatusSynced] = useState(false);
   const isPreviewCameraOn = useSelector(getIsPreviewCameraOn);
 
-  const localVideoStream = props.localParticipant?.videoStream;
-  const isVideoReady = localVideoStream?.isAvailable;
-  const renderElement = props.localParticipant.videoStream?.renderElement;
+  const videoStream = props.localParticipantVideoStream;
```

Comment:
Suggestion
For the sake of readability, can we continue the usage of `localParticipantVideoStream` variable throughout instead of assigning it to `videoStream` ?

Reply:
Leaving as is because that is a very wordy name for a local variable.

If our functions are so long that we need long variable names, we should be breaking those functions up with helpers. :)